spec/helpers/nav_helper_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe NavHelper, type: :helper do
```

Comment:
Could this tests be too brittle? Do we really need to test the classes that are generated?